gammapy/spectrum/utils.py

```diff
@@ -13,19 +16,254 @@
            'np_to_pha',
            ]
 
-# TODO: remove functions now that we have classes!
 
+def find_log_centers(bin_edges):
+    """Compute energy bin log centers.
+
+    Bin center = sqrt(low_edge * high_edge)
+    """
+    return np.sqrt(bin_edges[:-1] * bin_edges[1:])
+
+
+def find_log_edges(bin_centers):
+    """Compute equally log-spaced energy bin edges
+    """
+
+    bin_edges = np.sqrt(bin_centers[:-1] * bin_centers[1:])
+    first = bin_centers[0] * bin_centers[0] / bin_edges[0]
+    last = bin_centers[-1] * bin_centers[-1] / bin_edges[-1]
+    bin_edges = np.append(first, bin_edges)
+    bin_edges = np.append(bin_edges, last)
+
+    return bin_edges
+
+
+class EnergyBinning(object):
+
+    """Class to handle energy dimension for e.g. counts spectra, ARF tables
```

Comment:
Always start docstrings with a short single line ending with a dot.
This is then what gets displayed in overview lists of functions / classes:
https://gammapy.readthedocs.org/en/latest/irf/index.html#classes